src/gluonnlp/model/bert.py

```diff
@@ -1270,4 +1282,7 @@ def get_bert_model(model_name=None, dataset_name=None, vocab=None, pretrained=Tr
         ignore_extra = not (use_pooler and use_decoder and use_classifier)
         _load_pretrained_params(net, model_name, dataset_name, root, ctx, ignore_extra=ignore_extra,
                                 allow_missing=pretrained_allow_missing)
-    return net, bert_vocab
+    if tokenizer:
+        return net, bert_vocab, tokenizer
+    else:
+        return net, bert_vocab
```

Comment:
Shall we have a separate function to `gluonnlp.data.get_tokenizer` to download tokenizers for pre-trained model?

@leezu @szha @sxjscience what do you think

Reply:
@eric-haibin-lin how about the vocab, should it also be exposed as `gluonnlp.data.get_vocab`?

Reply:
That I am not sure - because currently all get_model calls returns both model and vocab. Having a get_vocab API leads to duplication. 
If we can change get_model API, I'd suggest we remove returned vocab from API, but that's not a sensible thing to do for v0.9.

What about using get_model to return model and vocab, get_tokenizer to return tokenizer in v0.9, and then in v1.0 we break  get_model into two functions for model and vocab separately ?

Reply:
I think that's a good  plan for providing `get_tokenizer()`  in v0.9.

Do we need to define class for getting model?  `self.get_model()`, `self.get_vocab()`, `self.get_tokenizer()` in v1.0